Automattic/newspack-blocks
# feat(ras-acc): add standalone newsletter signup form

### All Submissions:






### Changes proposed in this Pull Request:



Closes https://app.asana.com/0/1206943664367857/1205668937700578/f

This PR is paired with the following plugin PR: https://github.com/Automattic/newspack-plugin/pull/3120

This PR separates the post-checkout newsletters signup form from the checkout thank you modal.

This PR also moves logic for rendering and handling the newsletters signup form from the blocks plugin over to the main Newspack plugin since the form only renders whenever RAS is active. (Let me know if there is a preference to move this back to blocks)

![Screenshot 2024-05-20 at 16 51 15](https://github.com/Automattic/newspack-blocks/assets/17905991/a68aca28-eb01-4bcb-8e41-7bf1e9caa235)

### How to test the changes in this Pull Request:

See testing instructions in https://github.com/Automattic/newspack-plugin/pull/3120

### Other information:
